### PR TITLE
Port batch test.dbservlet and jbatch_fat projects to open-liberty

### PR DESCRIPTION
**Pull request by dazavala**
-Moves com.ibm.wsjbatch.test.dbservlet project to dir com.ibm.ws.jbatch.common_fat
  -Moves com.ibm.ws.jbatch_fat to com.ibm.ws.jbatch.open_fat
  -Adds gradle targets to common_fat that build app archives
  -Adds app utility class to common_fat that shrinkwraps and deploys app archives
  -Adds gradle targets to open_fat to import sources and app archives from common_fat
  -Enables open_fat to build common (shared) and local test archives locally
  -Modifies all open_fat test classes to hot-deploy required app archives
  -Repeats open_fat for ee9, only

### CODE REVIEW

**Comment by dazavala** [comment]
#build

**Comment by dazavala** [comment]
#build 
Moved jbatch_fat project to com.ibm.ws.jbatch.open_fat.

**Comment by dazavala** [comment]
#build
Ack...  Changed entry in test-applications/.gitignore to be specific regarding application file paths imported by gradle.  The previous entry caused git to ignore project test sources.

File: dev/com.ibm.ws.jbatch.open_fat/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+src: \
+	fat/src,\
+	test-applications/batchFAT.war/src,\
+	test-applications/batchFAT.war/resources,\
+	test-applications/batchSecurity.war/src,\
+	test-applications/batchSecurity.war/resources,\
+	test-applications/dbServletApp.war,\
+	test-applications/dbServletApp.war/resources,\
+	test-applications/BonusPayout.war,\
+	test-applications/BonusPayout.war/resources,\
```
Comment (cgianfra) [inline dev/com.ibm.ws.jbatch.open_fat/bnd.bnd:23]: Do we need these if we're building the applications via BatchAppUtils?

File: dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE9Action.java [hunk 2]
```diff
@@ -282,8 +282,8 @@ public static void transformApp(Path appPath, Path newAppPath) {
             args[0] = appPath.toAbsolutePath().toString(); // input
             args[1] = outputPath.toAbsolutePath().toString(); // output
 
-            args[2] = "-q"; // quiet output
-
+            //args[2] = "-q"; // quiet output
+            args[2] = "-v";
```
Comment (cgianfra) [inline dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE9Action.java:286]: Another old change from my branch that can be reverted

**Review by cgianfra — APPROVED** [review]

**Pull request merged**